### ISSUE TITLE
Re-organise asgardcms configuration files

### ISSUE THREAD

**Issue by nWidart** [AsgardCms/Platform#189]
At the time of the original development, laravel beta 5 didn't allow sub folders in the `config/` directory. Which was possible in laravel 4.

However now this is supported, in this regard I'd like to re-organise the asgardcms configuration file structure.

Option one, simple namespace:

Pros: already compatible with current calls to laravel config class.
Cons:

```
config/
    - asgardcms/
        - user
            - config.php
            - permissions.php
```

Option 2: add an advanced config folder, that contains asgardcms config files, that will not change often, or never.

Pros: separates advanced config that might cause confusion out of the module config folder
Cons: yet another big list of files in the `advanced` folder.
```
config/
    - asgardcms/
        - advanced/
            - user.permissions.php
        - user
            - config.php
```

**Comment by nWidart** [comment]
This is now implemented in the 2.0 branches.

**Comment by arcanedev-maroc** [comment]
Check also: https://github.com/AsgardCms/Platform/issues/156
